internal/plugins/ansible/v1/scaffolds/init.go

```diff
@@ -97,11 +97,11 @@ func (s *initScaffolder) scaffold() error {
 		&prometheus.Kustomization{},
 		&prometheus.ServiceMonitor{},
 
-		&manager.Manager{Image: imageName},
+		&manager.Config{Image: imageName},
```

Comment:
These changes are unrelated to flag name changes, can you put them in a separate PR?

Reply:
I need do the changes to be able to compare the files. 
I am just renaming the files and keeping then == upstream for we are able to diff